beacon-chain/db/BUILD.bazel

```diff
@@ -17,7 +17,7 @@ go_library(
         "validator.go",
     ],
     importpath = "github.com/prysmaticlabs/prysm/beacon-chain/db",
-    visibility = ["//beacon-chain:__subpackages__"],
+    visibility = ["//visibility:public"],
```

Comment:
Please revert. I think we did this for a temporary demo, but this is a no-no.

Reply:
+1, please revert this